packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx

```diff
@@ -93,7 +114,12 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
             </div>
           )}
         </div>
-
+        {benchmarkData && (
```

Comment:
it seems like it might be valuable to have targetData still display even if we have no benchmark. for the wrapping div we could just do (benchmarkData || targetData) && [ui code here]

Reply:
Change it to  (benchmarkData || targetData) && [ui code here]